### PR TITLE
chore: Brands Studio to purple and sets theme color to match

### PR DESCRIPTION
**Pull request by dthyresson**
This PR changes the favicon to purple and then rebrands the theme from orange to purple to match - which looks nice in dark mode and plays well with the Orbit community search.

* favicon generated
* index html updated

To make the look more consistent, it

* uses "tremor" classes to set text colors to respect light dark themed text (error messages, nav menu items)
* uses a svg icon for the StudioIcon so it can automatically be set with the theme

image

Now if we chose to pick a different color, then (apart from making a new favicon) can simply set the tremor theme colors in tailwind config.

---

Fixes https://github.com/redwoodjs/studio/issues/23
Fixes https://github.com/redwoodjs/studio/issues/31
^^ (some of, still could do UX review, but maybe make a new PR)

### CODE REVIEW

File: web/public/mark.svg
Comment (Tobbe) [inline web/public/mark.svg:1]: Why is this one so tiny now? And do we want it to be purple instead?

File: artwork/favicon-purple/android-chrome-512x512.png
Comment (Tobbe) [inline artwork/favicon-purple/android-chrome-512x512.png:1]: Can you use our svg as a base instead? The png looks very blurry when scaled up like this.

File: artwork/favicon/android-chrome-192x192.png
Comment (Tobbe) [inline artwork/favicon/android-chrome-192x192.png:1]: Do we need this orange version? What is it used for?

**Review by Tobbe — CHANGES_REQUESTED** [review]

**Comment by Tobbe** [comment]
I've made some changes to this PR:
* Updated the favicon images to be higher quality
* Used https://realfavicongenerator.net to generate images/icons and html code to best support various browsers and devices. (Highest ranking answers on StackOverflow ([1](https://stackoverflow.com/questions/48956465/favicon-standard-2024-svg-ico-png-and-dimensions), [2](https://stackoverflow.com/questions/25952907/what-is-the-best-practice-for-creating-a-favicon-on-a-web-site)) both point to that generator)

I used the attached SVG as a starting point when generating the icons. I then filled it with Tailwind's violet-700 color and added a drop shadow. Finally I centered the entire icon (including the drop shadow) in a 1000x1000px png file)

diecut_mark

**Comment by Tobbe** [comment]
What's left to do now before this can be merged is to change it back to use a larger icon for the logo in the top-left, to match what we already have for the current version
image

File: web/public/mark.svg
Comment (Tobbe) [reply web/public/mark.svg:1]: This is not used, so I removed it

File: artwork/favicon/android-chrome-192x192.png
Comment (Tobbe) [reply artwork/favicon/android-chrome-192x192.png:1]: It's not used, I removed it.

File: artwork/favicon-purple/android-chrome-512x512.png
Comment (Tobbe) [reply artwork/favicon-purple/android-chrome-512x512.png:1]: Fixed the image quality issues.

**Comment by dthyresson** [comment]
> What's left to do now before this can be merged is to change it back to use a larger icon for the logo in the top-left, to match what we already have for the current version
> image

Reworked markup and doesn't use Icon component:

image

**Comment by dthyresson** [comment]
Thoughts on using the svg that has the white stroke outline? I only saw pngs for that image.

**Comment by Tobbe** [comment]
> Thoughts on using the svg that has the white stroke outline? I only saw pngs for that image.

You can try if you want! :)  The .svg is attached to a reply here on this PR a few posts up. It has the red color, but should be easy enough to switch to violet

**Comment by dthyresson** [comment]
> > Thoughts on using the svg that has the white stroke outline? I only saw pngs for that image.
> 
> You can try if you want! :) The .svg is attached to a reply here on this PR a few posts up. It has the red color, but should be easy enough to switch to violet

Yes, I prefer that:

image

**Comment by Tobbe** [comment]
> Yes, I prefer that:

I agree. That looks nicer 🙂

**Comment by Tobbe** [comment]
@dthyresson I understand that you wanted to keep the title text aligned with the text for the menu items.

studio-icon-ss

But I thought it looked too cramped. So I decided to keep the spacing between icon and text consistent, instead of the alignment.

studio-dist

**Pull request merged**